File: jdroid-android-about/src/main/java/com/jdroid/android/about/SpreadTheLoveActivity.java
package com.jdroid.android.about;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class SpreadTheLoveActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return AboutAppModule.Companion.get().getAboutContext().getSpreadTheLoveFragmentClass();
	}
}

File: jdroid-android-about/src/main/java/com/jdroid/android/about/SpreadTheLoveActivity.kt
package com.jdroid.android.about

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class SpreadTheLoveActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return AboutAppModule.get().aboutContext.getSpreadTheLoveFragmentClass()!!
    }
}
